mapic/mistapiconnector_app.go

```diff
@@ -189,27 +189,27 @@ func (mc *mac) NukeStream(playbackID string) {
 }
 
 func (mc *mac) handleStreamBuffer(ctx context.Context, payload *misttriggers.StreamBufferPayload) error {
-	// We only care about connections ending
-	if !payload.IsEmpty() {
-		return nil
-	}
-
+	isActive := !payload.IsEmpty()
 	playbackID := payload.StreamName
 	if mc.baseStreamName != "" && strings.Contains(playbackID, "+") {
 		playbackID = strings.Split(playbackID, "+")[1]
 	}
 	if info, ok := mc.getStreamInfoLogged(playbackID); ok {
-		glog.Infof("Setting stream's manifestID=%s playbackID=%s active status to false", info.id, playbackID)
-		_, err := mc.lapi.SetActive(info.id, false, info.startedAt)
+		glog.Infof("Setting stream's manifestID=%s playbackID=%s active status to %v", info.id, playbackID, isActive)
+		_, err := mc.lapi.SetActive(info.id, isActive, info.startedAt)
 		if err != nil {
 			glog.Error(err)
```

Comment:
Should we add some error context here? It looks like we don't get much from the SetActive method:

```go
func (lapi *Client) SetActive(id string, active bool, startedAt time.Time) (ok bool, err error) {
        if id == "" {
                return false, errors.New("empty id")
        }
```

Reply:
Added in 5075f8fbf25e889db5a250e811f88a6ed5f6d324